<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PictureSearchWidget</class>
 <widget class="QWidget" name="PictureSearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search Web Pictures</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="DottedLineEdit" name="lineEdit"/>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="searchButton"/>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QRadioButton" name="fRadio">
       <property name="text">
        <string>Flickr</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="gRadio">
       <property name="text">
        <string>Google Images</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="googleOptions" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QComboBox" name="contentCombo">
          <item>
           <property name="text">
            <string>Any Content</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>News</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Faces</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Photos</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Clipart</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Lineart</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="sizeCombo">
          <item>
           <property name="text">
            <string>Any Size</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Medium</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Large</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Icon</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="PicturesListWidget" name="listWidget">
     <property name="minimumSize">
      <size>
       <width>450</width>
       <height>250</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragDrop</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>75</height>
      </size>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
     <property name="gridSize">
      <size>
       <width>105</width>
       <height>80</height>
      </size>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PicturesListWidget</class>
   <extends>QListWidget</extends>
   <header>QListWidget</header>
  </customwidget>
  <customwidget>
   <class>DottedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>QLineEdit</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>searchButton</tabstop>
  <tabstop>listWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
